src/components/menus/directory-tree-contextual-menu.tsx

```diff
@@ -261,6 +262,18 @@ export default function DirectoryTreeContextualMenu(props: Readonly<DirectoryTre
                     messageDescriptorId: 'createNewFilter',
                     callback: () => handleOpenDialog(DialogsId.ADD_NEW_FILTER),
                     icon: <AddIcon fontSize="small" />,
+                    subMenuItems: [
+                        {
+                            messageDescriptorId: 'createNewExplicitNamingFilter',
+                            callback: () => handleOpenDialog(DialogsId.ADD_NEW_EXPLICIT_NAMING_FILTER),
+                            icon: <AddIcon fontSize="small" />,
+                        },
+                        {
+                            messageDescriptorId: 'createNewExpertFilter',
+                            callback: () => handleOpenDialog(DialogsId.ADD_NEW_EXPERT_FILTER),
+                            icon: <AddIcon fontSize="small" />,
```

Comment:
```suggestion
                        },
                        {
                            messageDescriptorId: 'createNewExpertFilter',
                            callback: () => handleOpenDialog(DialogsId.ADD_NEW_EXPERT_FILTER),
```
I think the `AddIcon` shouldn't be included : they are not in the specs and the `AddIcon` on the left is enough to type the action.

Reply:
YES, I think icon is not needed here

Reply:
I don't think this is better now : 
![sou-menu](https://github.com/user-attachments/assets/c0ea621f-55c6-44c2-9462-ef0e02f83c42)

I think there shouldn't be any icon, like in the specs (and in gridstudy).

Reply:
DONE : https://github.com/gridsuite/gridexplore-app/pull/647/commits/0f3aac4e5309781a74906ea4a20bac04a08b7597

![image](https://github.com/user-attachments/assets/d0ea5a71-d721-4253-8fc8-162d02c1ffd3)